### PR TITLE
Demo enhancements

### LINKED ISSUE #1: Features
Instead of the preview given in food/index.html, show the full data set that will be exported (or top 20 entries for speed, whatever), in the way that this does:

https://bl.ocks.org/syntagmatic/raw/482706e0638c67836d94b20f0cb37122/

- [ ] ability to bring axes back into view after dragging off screen (checkboxes for show/hide rather than drag offscreen is more than acceptable) 

- [ ]  preview export

- [ ] disable scrolling. Only interference with mobile compatibility is that you can accidentally scroll the page instead of dragging on a column. 

- [x]  be able to load in arbitrary column names from csv (seems already possible but code crashes because of hard-coded column names for categories, which we do not need anyway).
- [x] categories section has no relevant counterpart in our dataset. Get rid of it entirely, it’ll never be relevant for what we are doing. The equivalent filtering would have already been performed in the computations by the time we’re visualizing this dataset.
- [x] keep dark mode, keep/exclude buttons, export, 
- [x]  rotate column labels 45°

### PR DESCRIPTION
**Pull request by elliottregan**
Closes #1

### CODE REVIEW

File: food/parallel.js [hunk 1]
```diff
@@ -123,13 +122,6 @@ d3.csv("nutrients.csv", function(raw_data) {
           g.attr("transform", function(d) { return "translate(" + position(d) + ")"; });
           brush_count++;
           this.__dragged__ = true;
-
-          // Feedback for axis deletion if dropped
```
Comment (mathematicalmichael) [inline food/parallel.js:127]: is this removing the ability to remove an axis?

File: food/parallel.js [hunk 2]
```diff
@@ -143,11 +135,6 @@ d3.csv("nutrients.csv", function(raw_data) {
             var extent = yscale[d].brush.extent();
           }
 
-          // remove axis if dragged all the way left
-          if (dragging[d] < 12 || dragging[d] > w-12) {
-            remove_axis(d,g);
```
Comment (mathematicalmichael) [inline food/parallel.js:148]: ahh looks like it. can we make this into a `hide_axis` instead? or does D3 not have a sense of invisible objects?

File: food/parallel.js [hunk 3]
```diff
@@ -692,18 +593,9 @@ function exclude_data() {
   rescale();
 }
 
-function remove_axis(d,g) {
```
Comment (mathematicalmichael) [inline food/parallel.js:695]: man... what is happening here?

File: food/parallel.js (hunk 2 above)
Comment (elliottregan) [reply food/parallel.js:148]: not that i know of. we would need to keep track of these object separately from D3 for reference later. When there is time, a thorough refactor is needed.

File: food/parallel.js (hunk 1 above)
Comment (elliottregan) [reply food/parallel.js:127]: yes. This block specifically is handling the style (the red rect), and a block somewhere else is responsible for removing it.

Comment (mathematicalmichael) [reply food/parallel.js:127]: red rectangle?

File: food/parallel.js (hunk 3 above)
Comment (elliottregan) [reply food/parallel.js:695]: remove 1 dimension, rescale the domain for the shorter length of dimensions, shifting the existing ticks lines, removing the dimension from the the SVG (the ticks), and finally re-rendering D3.

Comment (mathematicalmichael) [reply food/parallel.js:695]: oh.... okay, I see now how that's happening. the tick lines are objects themselves that have to be placed on top of vertical lines. wow. of course. I get how this renders now... thank you for your patience in translating that.

**Pull request merged**